.github/workflows/build.yml

```diff
@@ -0,0 +1,61 @@
+name: Build and run tests
+on: [push, pull_request]
+env:
+  app_id: pvr.zattoo
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
> Not sure if it matters: Is it ok to have "matrix" here instead of "nexus"?

There you must aware, is hard as there currently Kodi has as name matrix. This name also used for some other things (not Kodi related :sweat_smile:).

Is related to named os in this data matrix.

Reply:
Oh, stupid me. Thank you :)